ocdaction/ocdaction/templates/challenge/challenge_score_form.html

```diff
@@ -41,7 +41,9 @@
                 {% endif %}
             {% endfor %}
 
-            <!---button class="btn btn-primary pull-left col-xs-4 col-xs-offset-5" type="submit">Complete the Challenge</button-->
+            {% if anxiety_score_form.score_after_120_min.value %}
+                <button class="btn btn-primary pull-left col-xs-4 col-xs-offset-5" type="submit">Complete the Challenge</button>
```

Comment:
It seems we're missing a url here?

Reply:
Yes sorry - will change now - for some reason I forgot we had a challenge complete page already!
With regards to the heroku deployment I think you will have to drop the database and recreate as on here.  The change was quite significant in that the models were set up so that you could have multiple score cards for each challenge, which needed changing to a one to one relationship.